### PR TITLE
Task/APPS-3250 — update homepage

### PR DESCRIPTION
**Pull request by giuliacampana**
Screen Shot 2021-11-19 at 5 28 04 PM
Screen Shot 2021-11-19 at 5 28 18 PM
Screen Shot 2021-11-19 at 5 28 25 PM
Screen Shot 2021-11-19 at 5 28 32 PM
Screen Shot 2021-11-19 at 5 28 39 PM
Screen Shot 2021-11-19 at 5 28 46 PM
Screen Shot 2021-11-19 at 5 28 56 PM
Screen Shot 2021-11-19 at 5 29 02 PM
Screen Shot 2021-11-19 at 5 29 10 PM

### CODE REVIEW

**Review by UnsungHero97 — COMMENTED** [review]
Can you run `laptop.png` through https://tinypng.com?

File: src/containers/home/HomePage.js [hunk 1]
```diff
@@ -27,133 +35,169 @@ import {
...
+            <LearnMoreButton href={PRODUCTS_CHRONICLE} />
+          </Tile>
+          <Tile align={{ h: 'start' }}>
+            <LinkCircleIcon width="50" />
+            <Typography textAlign="left" variant="h6">
+              Real-time Insights
+            </Typography>
+            <Typography textAlign="left" variant="body1">
+              {`Advance your research using our ecological momentary assessment features to measure participant's
+              behaviors and experiences in real-time.`}
```
Comment (UnsungHero97) [inline src/containers/home/HomePage.js:128]: Can you make sure the new line here isn't being rendered in the browser?

File: src/containers/home/HowItWorksSection.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+          <Typography color={NEUTRAL.N400} fontWeight={600} maxWidth={{ lg: 420 }} uppercase variant="body1">
+            Step 3
+          </Typography>
+          <Typography maxWidth={{ lg: 420 }} variant="h3">
+            Clean
+          </Typography>
+          <Typography maxWidth={{ lg: 480 }} variant="body2">
+            {`Now onto the fun stuff - you’ve collected your data and all that’s left is to make sense of it!
+              Chronicle’s library of scripts and support tools help improve the speed and accuracy of cleaning,
+              structuring, and validating your data, so you can spend more time on the analysis.`}
```
Comment (UnsungHero97) [inline src/containers/home/HowItWorksSection.js:71]: Same comment about new lines.

File: src/containers/home/HomePage.js (hunk 1 above)
Comment (giuliacampana) [reply src/containers/home/HomePage.js:128]: the new line is not being rendered in the browser!

File: src/containers/home/HowItWorksSection.js (hunk 2 above)
Comment (giuliacampana) [reply src/containers/home/HowItWorksSection.js:71]: new lines here also not being rendered!

**Review by UnsungHero97 — APPROVED** [review]

**Review by geekbeast — APPROVED** [review]

File: src/containers/home/ComplianceAndSecuritySection.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+const LogosWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  @media only screen and (max-width: ${MEDIA_QUERY_SM}px) {
+    flex-direction: column;
+  }
```
Comment (solomontang) [inline src/containers/home/ComplianceAndSecuritySection.js:33]:
```suggestion
```
default flex-direction looks better imo
image

**Review by anzioka — APPROVED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Pull request merged**